includes/admin/upgrades/upgrade-functions.php

```diff
@@ -2660,6 +2664,65 @@ function give_v220_upgrades(){
 	give_v220_delete_wp_session_data();
 }
 
+/**
+ * Version 2.3.0 automatic updates
+ *
+ * @since 2.3.0
+ */
+function give_v230_upgrades(){
```

Comment:
@ravinderk update function name

Reply:
@DevinWalker thank you for catching that